command/server.go

```diff
@@ -534,6 +532,16 @@ func (c *ServerCommand) runRecoveryMode() int {
 		}
 	}
 
+	raftStorage, ok := backend.(*raft.RaftBackend)
+	if ok {
+		if err := raftStorage.StartRecoveryCluster(context.Background(), raft.Peer{
```

Comment:
I don't think we should do this until the recovery token has been created, otherwise there isn't any authorization prior to resizing the cluster.

Reply:
That's a very valid point.